### PR TITLE
Implement ES6 features in awesome-books project

### PR DESCRIPTION
**Pull request by bienvenuushindi**
#### Re-create Awesome books app following the rules:
- Use ES6 syntax.
- Divide code into modules. 
- Save each module in a separate file in the modules dir. 
- Import modules in the index.js file. 
- Refactor your methods and functions to arrow functions.
- Use let and const in the correct way.
#### Display the current date below the navigation bar.
- use  [Luxon](https://moment.github.io/luxon/#/install?id=es6) library
- Import it as a module.

### CODE REVIEW

File: modules/book.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                         <span>&nbsp; by &nbsp;</span>
+                         <div class='book-author mr-1'> ${book.author} </div>
+                    </div>
+                    <div class='action '>
+                        <button type='button' class='remove-button btn btn-danger ml-auto small btn-sm' id='${index}'>Remove</button>
+                    </div>
+                    <hr>
+                </li> `).join(' ');
+  }
+}
```
Comment (algerina) [inline modules/book.js]: - [ ] You did a good job so far, however, according to the requirement you have to divid your code into reasonable modules, the more you have, the best is.

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @bienvenuushindi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks good:heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/book.js (hunk 1 above)
Comment (bienvenuushindi) [reply modules/book.js]: @algerina Thanks for that remark, I appreciate it But what you are saying is somewhat confusing to me. Your comments aren't clear enough on why I'm going to have to modularize my Book class

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 

## Hi @bienvenuushindi 👋, 

Well done implementing the requested changes 👏
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Highlights 🎯 
no linter errors✔️
descriptive PR✔️
Good UI design✔️

### Optional Suggestions

- [x] Please consider correcting the success response a user sees when they add a new book. See below 👇 
image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**